specificatie/openapi.yaml

```diff
@@ -0,0 +1,377 @@
+openapi: 3.0.0
+servers:
+  - description: "APILAB testserver"
+    url: https://www.haalcentraal.nl/haalcentraal/api/brp
+info:
+  title: Bevragingen historie ingeschreven personen
+  description: |
+                API voor het ontsluiten van historische gegevens in relatie tot ngeschreven personen en aanverwante gegevens uit de GBA en RNI. Met deze API worden de historische gegevens van ingeschreven personen, hun kinderen, partners en ouders ontsloten.
```

Comment:
> Met deze API worden de historische gegevens van ingeschreven personen, hun kinderen, partners en ouders ontsloten

Hoe worden in deze API historische gegevens van kinderen en ouders ontsloten? Alleen heel specifieke historische gegeven van personen worden ontsloten
Waarom niet direct zeggen wat deze API levert: historie van verblijfplaats, partner, verblijfstitel en nationaliteit

Reply:
Aangepast